bwa/src/game.cpp

```diff
@@ -1,32 +1,47 @@
+#include <exception>
 #include <string>
 #include <utility>
 #include "game.hpp"
 
 constexpr const char* WINDOW_TITLE = "Bubble Warrior Adventures!";
 
 bwa::Game::Game() {
+	// Loads the config into the sol::state
 	_lua.script_file("config.lua");
+	// Converts the config's x and y resolution into a table
 	sol::table resolution = _lua["resolution"];
-	_font.loadFromFile(_lua["game_fonts"]["normal"]);
+	// Checks for font file and sets _font to it
+	if (!_font.loadFromFile(_lua["game_fonts"]["normal"]))
+	{
+		throw std::invalid_argument(_FILE_NOT_FOUND_ERROR + _lua["game_fonts"]["normal"].get<std::string>());
```

Comment:
Change to this because it's too long

```cpp
throw std::invalid_argument(_FILE_NOT_FOUND_ERROR +
	_lua["game_fonts"]["normal"].get<std::string>());
```